src/metabase/server/handler.clj

```diff
@@ -42,11 +43,22 @@
        (name kkey))
      response output-stream)))
 
+
+(defn wrap-locale-binding
```

Comment:
This is very POC, it should not live in this file but I'll let some proper BE dev decide where it should live